alacritty/src/display.rs

```diff
@@ -240,6 +238,7 @@ impl Display {
 
         info!("Cell size: {} x {}", cell_width, cell_height);
         info!("Padding: {} x {}", size_info.padding_x(), size_info.padding_y());
+        info!("Width: {}, Height: {}", size_info.width(), size_info.height());
```

Comment:
We previously haven't logged this at startup, but I think it is very useful especially because we log it on resize.